py-polars/polars/internals/lazyframe/frame.py

```diff
@@ -1671,11 +1674,13 @@ def select(
         """
         if exprs is None and not named_exprs:
             raise ValueError("Expected at least one of 'exprs' or **named_exprs")
-        elif exprs is None:
-            exprs = []
```

Comment:
These lines were not necessary - this is taken care of when `exprs` is passed to `selection_to_pyexpr_list`.